packages/style/scss/controls/flat-select.scss

```diff
@@ -92,66 +84,37 @@
                 }
             }
         }
-
-        .icon {
-            margin-bottom: -2px;
-        }
     }
 
-    &.mod-btn-group .flat-select-option {
-        height: $button-height;
-        padding: $button-padding-y $button-padding-x;
-        font-weight: var(--main-font-bold);
+    &.mod-btn-group {
+        --gap: 0px;
+        border-radius: 8px;
 
-        font-size: $button-font-size;
-        text-transform: uppercase;
-        border-radius: 0;
+        .flat-select-option {
+            @extend .flat-select-option;
```

Comment:
it extends itself?

Reply:
kinda? 🙈  the mod-group wrap the options in a div and I need to use the same class to style the options individually 😆